requirements.txt

```diff
@@ -1,12 +1,44 @@
-Flask==0.10.1
-Jinja2==2.7.3
-MarkupSafe==0.23
-Werkzeug==0.9.6
-gnureadline==6.3.3
-itsdangerous==0.24
-rauth==0.7.0
-requests==2.3.0
-wsgiref==0.1.2
-
-gunicorn==18.0
-Flask-SSLify==0.1.4
+aniso8601==7.0.0
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-29361   
**Check Name:**    
**Severity:** CRITICAL   
**Fixed Version:** 2.1.1   
**Description:** ** DISPUTED ** Improper parsing of HTTP requests in Pallets Werkzeug v2.1.0 and below allows attackers to perform HTTP Request Smuggling using a crafted HTTP request with multiple requests included inside the body. NOTE: the vendor's position is that this behavior can only occur in unsupported configurations involving development mode and an HTTP server from outside the Werkzeug project.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-29361